TFW-3122 - fix to US page scroll

jasonhale opened the pull request:
Introduced a vertical scrollbar placement problem with TFW-2941.  This fixes that issue.

Screenshot of initial prob:
![Scrollbar on US](https://user-images.githubusercontent.com/3496073/54140211-18935980-43f1-11e9-9205-a1d2067c636e.png)

troyblank commented on assets/sass/base/_mixins.scss:
```diff
@@ -64,7 +64,7 @@
     position: relative;
     width: 100%;
     max-width: $max-content-width;
-    height: 100%;
```
This causes the issue that the welcome graphic won't sit at the bottom of the screen.

![Screen Shot 2019-03-11 at 12 28 15 PM](https://user-images.githubusercontent.com/996725/54140132-572d1200-43f9-11e9-8d5d-a7bee9f1a71f.png)

jasonhale replied:
Awesome, thanks

jasonhale commented:
@troyblank - ok, updated.  Thanks for catching.  I reverted the style changes introduced into the splash page component (also, it would've modified the splash as it is on Workflow), and then added a container in US to get it layed out correctly.

troyblank commented:
@jasonhale sorry man just one more small area it's not flush to the bottom.

![Screen Shot 2019-03-11 at 2 21 23 PM](https://user-images.githubusercontent.com/996725/54147658-09b8a100-4409-11e9-88db-930c3605c349.png)

jasonhale commented:
No prob.  Updated.
